### PR TITLE
📖 Add v1.6 Release Team members' GitHub/Slack handles

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it**:
- Fixes formatting issues in the table
- adds v1.6 Release Team members GitHub/Slack handles in an alphabetical order

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/release/releases/release-1.6.md [hunk 1]
```diff
@@ -4,33 +4,33 @@
 
 The following table shows the preliminary dates for the `v1.6` release cycle.
 
-| **What**                                             | **Who**      | **When**                   | **Week** |
-|------------------------------------------------------|--------------|----------------------------|----------|
-| Start of Release Cycle                               | Release Lead | Monday 31st July 2023      | week 1   |
+| **What**                                             | **Who**      | **When**                    | **Week** |
```
Comment (sbueringer) [inline docs/release/releases/release-1.6.md:7]: Thx for this fix. I didn't want to annoy anyone with nitpicking when the KubeCon line was added :)

File: docs/release/releases/release-1.6.md [hunk 2]
```diff
@@ -4,33 +4,33 @@
...
+| **Role**                                  | **Lead** (**GitHub / Slack ID**)                                                          | **Team member(s) (GitHub / Slack ID)** |
 |-------------------------------------------|-------------------------------------------------------------------------------------------|----------------------------------------|
-| Release Lead                              | TBD | TBD                                    |
-| Communications/Docs/Release Notes Manager | TBD | TBD                                    |
-| CI Signal/Bug Triage/Automation Manager   | TBD | TBD                                    |
-| Maintainer                                | TBD | TBD                                    |
\ No newline at end of file
+| Release Lead                              | Furkat Gofurov ([@furkatgofurov7](https://github.com/furkatgofurov7) ✉️ ️`@Furkat Gofurov`) | Chirayu Kapoor ([@chiukapoor](https://github.com/chiukapoor) ✉️ `@Chirayu Kapoor`) <br> Prajyot Parab ([@Prajyot-Parab](https://github.com/Prajyot-Parab) ✉️ ️`@Prajyot Parab`) <br> Stephen Cahill ([@cahillsf](https://github.com/cahillsf) ✉️ ️`@Stephen Cahill) |
```
Comment (sbueringer) [inline docs/release/releases/release-1.6.md:33]:
```suggestion
| Release Lead                              | Furkat Gofurov ([@furkatgofurov7](https://github.com/furkatgofurov7) ✉️ ️`@Furkat Gofurov`) | Chirayu Kapoor ([@chiukapoor](https://github.com/chiukapoor) ✉️ `@Chirayu Kapoor`) <br> Prajyot Parab ([@Prajyot-Parab](https://github.com/Prajyot-Parab) ✉️ ️`@Prajyot Parab`) <br> Stephen Cahill ([@cahillsf](https://github.com/cahillsf) ✉️ ️`@Stephen Cahill`) |
```

**Comment by sbueringer** [comment]
Just nit otherwise lgtm

/assign @vincepri 
(as the "maintainer" :joy:)

File: docs/release/releases/release-1.6.md (hunk 2 above)
Comment (furkatgofurov7) [reply docs/release/releases/release-1.6.md]: thanks for catching it, fixed now

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Comment by sbueringer** [comment]
Just ignore it. We have this across multiple repos and I noticed that the action doesn't block Tide from merging

**Comment by furkatgofurov7** [comment]
> You can ignore it. We have this across multiple repos and I noticed that the action doesn't block Tide from merging

It is still not letting it in 🤷🏼‍♂️ 

/retest

**Comment by sbueringer** [comment]
Hm yeah. Can you commit --amend & push --force?

Not sure what's going on with the merge pool

**Comment by furkatgofurov7** [comment]
> Hm yeah. Can you commit --amend & push --force?
> 
> Not sure what's going on with the merge pool

> It has already cla label but EasyCLA looks to be stuck for some unknown reason.. perhaps I need to force push?

I already did, but force pushed it again now, let's see. Looks like a prow issue to me

**Comment by sbueringer** [comment]
The strang thing is: "Tide in merge pool".

I merged several PRs today where the GitHub action was pending, that was not an issue (btw there is the upstream issue for taht: https://github.com/communitybridge/easycla/issues/4071)

**Comment by sbueringer** [comment]
But looking at the Prow UI. I don't see a merge pool for CAPI including this PR.

Can you reopen this PR? (close this one and create a new separate one)

**Comment by sbueringer** [comment]
Interesting. Looks like this blocked it still somehow

**Comment by furkatgofurov7** [comment]
> /easycla

so this was the trick? sorry missed your comments. Thanks anyways for help!

**Comment by sbueringer** [comment]
> > /easycla
> 
> so this was the trick? sorry missed your comments. Thanks anyways for help!

No not really. Not sure what exactly was going on. I was able to merge PRs in other repos which were in the exact same state. The only thing different here was the "Tide in merge pool" thing. But at this stage CLA shouldn't block afaik.

I just retriggered the cla as they mentioned on https://github.com/communitybridge/easycla/issues/4071 that they should fixed the CLA issue

**Review by killianmuldoon — COMMENTED** [review]
/area release

**Pull request merged**